### ISSUE TITLE
py-multihash requires base58 which no longer supports python2

### ISSUE THREAD

**Issue by CMCDragonkai** [NixOS/nixpkgs#95731]
**Describe the bug**

Master branch `py-multihash` fails to build.

```
ERROR: Could not find a version that satisfies the requirement base58<2.0,>=1.0.2 (from py-multihash==0.2.3) (from versions: none)
ERROR: No matching distribution found for base58<2.0,>=1.0.2 (from py-multihash==0.2.3)
```

This can be fixed as soon as py-multihash bumps up its version of base58: https://github.com/multiformats/py-multihash/pull/8

In the mean time, the version constraint needs to be removed and loosened with a `postPatch` and `substituteInPlace`.

**Comment by CMCDragonkai** [comment]
```
    py-multihash = super.py-multihash.overridePythonAttrs (attrs: {
      postPatch = ''
        substituteInPlace setup.py --replace "'base58>=1.0.2,<2.0'," "'base58',"
      '';
    });
```

**Comment by CMCDragonkai** [comment]
Still waiting on upstream merge...

**Comment by CMCDragonkai** [comment]
It's been merged into upstream. So someone just needs update PyPi.

**Comment by Luflosi** [comment]
Closing, as the package was updated in #101801 and builds fine for me. I'm marking it as not broken in #112410.
